docs/terraform/terraform-provider.md

```diff
@@ -49,3 +49,27 @@ provider "harvester" {
 More details about the provider-specific configurations can be found in the [docs](https://registry.terraform.io/providers/harvester/harvester/latest/docs).
 
 Github Repo: [https://github.com/harvester/terraform-provider-harvester](https://github.com/harvester/terraform-provider-harvester)
+
+### Resource Timeouts
+
+There are several operations on resources that may potentially take some time to complete,
+e.g. creating a new image and downloading its content from the web. Depending on the hardware
+of the hosts and other factors, these operations may exceed default timeout settings, resulting
+in errors. To adjust timeouts of operations on resources, define a timeout block in the resource:
```

Comment:
```suggestion
Several resource-related operations (for example, creating a new image and downloading its content from the internet) may take some time to complete. Depending on the host hardware and other factors, these operations may exceed default timeout settings and cause errors. To modify timeout values for such operations, define a timeout block in the resource.
```

Reply:
Thanks for the suggestion, it's much better.
I've updated the PR.